src/wp-admin/css/forms.css

```diff
@@ -854,7 +854,7 @@ table.form-table td .updated p {
 	margin-bottom: 0;
 }
 
-.new-application-password-notice.notice input.code {
+.password-display input.code {
```

Comment:
Thinking this selector might be a bit generic to have in a global css file.

Reply:
Further namespaced it via https://github.com/WordPress/wordpress-develop/pull/638/commits/87550e27fa227c8b248489bea6f14a31700cb12a

Reply:
Or would you rather have an extra class on the input?

Reply:
That fix looks good to me.